### ISSUE TITLE
Reader and writer for Matlab 7.3 files.

### ISSUE THREAD

**Issue by aylakhan** [SCIInstitute/SCIRun#894]
Add support for reading [Matlab 7.3 files](http://www.mathworks.com/help/matlab/import_export/mat-file-versions.html). Look into export capabilities for fields and matrices. Requires HDF5 library.

**Comment by dcwhite** [comment]
@jessdtate Has this one changed in priority? How long will Matlab support their old file format?

**Comment by jessdtate** [comment]
I don't think it has changed.  I also don't think that matlab plans to stop supporting the old formats anytime soon.

**Comment by dcwhite** [comment]
**Triage 2026-06:** MATLAB 7.3 file reader/writer (HDF5-based) — MATLAB 7.3 format uses HDF5 internally. HDF5 is already a Superbuild dependency. Adding a `.mat` v7.3 reader/writer that uses `libhdf5` directly would be medium effort. The existing MATLAB IO code (pre-7.3) is in `src/Modules/Legacy/DataIO/`. Priority depends on whether users need large matrices that require v7.3 format.
